### PR TITLE
changed class snippets into playground 

### PR DESCRIPTION
**Pull request by i-walker**
This reduces overhead for ANK, as most classes and typeclasses are rewriten multiple times. 
This reduces this error, we saw in [1.6](https://github.com/arrow-kt/Category-Theory-for-Programmers.kt/pull/38#issuecomment-475945254). We may standardize this approach (to compile typeclasses within a playground environment) for all typeclasses, if so I will add it to the README.md accordingly.

### CODE REVIEW

**Comment by i-walker** [comment]
the commit messages are old :)

File: src/main/ank/1.7-functors-Part1.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+```
+```kotlin
+identity(x) == x
+```
+................
+```Haskell
+fmap id = id
+```
+```kotlin
+fmap{ s:A -> identity(s) } == identity
```
Comment (pakoito) [inline src/main/ank/1.7-functors-Part1.md]: respect whitespace

File: src/main/ank/1.7-functors-Part2.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+fun <A, B> fmap(): ((A) -> B, Kind<ForListK, A>) -> Kind<ForListK, B> =
+    { f, list -> ListK.functor().lift(f)(list) }
+```
+................
+```Haskell
+instance Functor List where
+    fmap _ Nil = Nil
+    fmap f (Cons x t) = Cons (f x) (fmap f t)
+```
+```kotlin:ank:playground
```
Comment (pakoito) [inline src/main/ank/1.7-functors-Part2.md]: I'm not sure if this @extension will work on the playground.

Comment (i-walker) [reply src/main/ank/1.7-functors-Part2.md]: Yes I forgot the import

**Comment by i-walker** [comment]
@pakoito Is it fine to merge?

**Comment by pakoito** [comment]
Done!

**Pull request merged**